mgmtfn/k8splugin/driver.go

```diff
@@ -354,6 +354,12 @@ func getEPSpec(pInfo *cniapi.CNIPodAttr) (*epSpec, error) {
 	resp.Tenant = tenant
 	resp.Network = netw
 	resp.Group = epg
+
+	// non-system pod with no EPG ? configure it in namespace
+	if pInfo.K8sNameSpace != "kube-system" && len(resp.Group) <= 0 {
+		resp.Group = pInfo.K8sNameSpace
```

Comment:
this may accidentally drop a pod into a pre-existed epg, unless we could make the epg creating only via k8s namespace and ingress.
or can we have a prefix for epgs come from k8s namespace ?